src/index.html

```diff
@@ -7,8 +7,34 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="image" href="index.html"><img src="images/logo.png" alt="Moyo logo"></a>
+
+      <nav class="nav">
+        <ul class="liAll">
+          <li class="li"><a class="nav__link is-active" href="/">Apple</a></li>
+          <li class="li"><a class="nav__link" href="/">Samsung</a></li>
+          <li class="li"><a class="nav__link" href="/">Smartphones</a></li>
+          <li class="li">
```

Comment:
```suggestion
        <ul class="liAll">
          <li class="li">
            <a class="nav__link is-active" href="/">
              Apple
            </a>
          </li>
```
also check code style 
<img width="953" alt="image" src="https://user-images.githubusercontent.com/102411130/202689373-536aae90-b5d3-4bb0-a06c-82d40be51945.png">

Reply:
and fix links
`href="/#apple"`

Reply:
Corrected